.github/workflows/ci.yml

```diff
@@ -102,6 +103,11 @@ jobs:
         uses: actions/setup-go@v5
         with: { go-version: "${{ matrix.language_version }}" }
 
+      - if: matrix.language == 'rust'
+        uses: dtolnay/rust-toolchain@master
```

Comment:
This _should_ fail, as I think we deny all non-github actions on our CI, for supply chain reason. I even if the action is compromised, our CI _shouldn't_ be able to do anything bad, as we only have sensitive permissions on the main branch, but, still. 


The good news is that GitHub runners come with rustup pre-installed, so we can use it directly here. I am a huge fan of `rustup` by the way, great tool ;-)

So I think we can do

```
- run: rustup default ${{ matrix.language_version }} && rustup component add clippy
```

or whatever is the right syntax.

Reply:
Done.